echoshader/echogram.py

```diff
@@ -118,7 +115,7 @@ class Echogram(param.Parameterized):
 
         panel.Row(echogram.widgets, echogram.view_gram)
 
-    """
+    """  # noqa
```

Comment:
I am just curious why we need `noqa` here. Does it not like long docstring section?

Reply:
Some lines of the docstring is >100 characters and causing flake8 to fail, so I added `noqa` here.

Reply:
Ah interesting I didn't know `noqa` can work with the entire block too!